test/e2e/step_definitions/generic.step.js

```diff
@@ -291,6 +291,14 @@ defineSupportCode(({ Given, When, Then, setDefaultTimeout }) => {
     optionElem.click().then(callback).catch(callback);
   });
 
+  Then('I click on "{className}" element no. {index}', (className, index, callback) => {
+    browser.sleep(500);
+    const optionElem = element.all(by.css(className)).get(index - 1);
```

Comment:
To be consistent with other steps, `className` should not contain the dot and be kebab-case instead of cammelCase.
https://github.com/LiskHQ/lisk-hub/blob/cd8441e93079f8e9ad3f887df9386d3d8246245a/test/e2e/step_definitions/generic.step.js#L147